modules/nextflow/src/main/groovy/nextflow/scm/AssetManager.groovy

```diff
@@ -730,6 +755,10 @@ class AssetManager {
     }
 
 
+    private static boolean isRemoteBranch(Ref ref) {
```

Comment:
Likely a unit test can be added for this

Reply:
Added a direct test in `AssetManagerTest.groovy`, and modified `CmdInfoTest.groovy` to test indirectly.